packages/console/src/pages/UserDetails/UserSettings/components/UserSsoIdentities/index.tsx

```diff
@@ -47,15 +47,16 @@ function UserSsoIdentities({ ssoIdentities }: Props) {
           const {
             providerLogo,
             providerLogoDark,
+            branding,
             connectorName: name,
           } = data.find((ssoConnector) => ssoConnector.id === identity.ssoConnectorId) ?? {};
           const { identityId: userIdentity, issuer } = identity;
 
-          if (!(providerLogo && providerLogoDark && name)) {
+          if (!(providerLogo && providerLogoDark && branding && name)) {
```

Comment:
Also this is a bit odd, it seems if any 1 of the 4 elements is missing, the return value will be undefined.

Reply:
Right.